apps/demo-react-vite/src/App.tsx

```diff
@@ -4,13 +4,15 @@ import viteLogo from "/vite.svg";
 import "./App.css";
 
 import { NativeAssetId, encrypt, decrypt } from "fuels";
+import * as asm from "@fuels/vm-asm";
 
 function App() {
   const [count, setCount] = useState(0);
 
   const { log } = console;
 
   log("Hello Fuels", NativeAssetId, encrypt, decrypt);
+  log("Hello ASM", asm);
```

Comment:
This was to validate that the first experiments were compiling correctly with these demos (vite, nextjs, cra).

But since all asm-related things are on the `fuel-vm` repo, it might be redundant.

Reply:
I agree. I don't think we (or any users) would be directly consuming this wasm package from a frontend anyways.

Reply:
To clear this out — this package will be embedded and used by the SDK, so regardless of whether users install it directly, it should work on the browser as this is how we will build things like the multi-call script and stuff like that.

When I say this may be redundant, it is because most validations are done outside of the TS SDK, as I tried to outline in this other [comment](https://github.com/FuelLabs/fuels-ts/pull/1080#discussion_r1272105691). Having said that, if we all agree it is good to have an extra validation layer with things like @camsjams suggested, I can happily do it.

Suggestions are welcomed!